### DISCUSSION TITLE
Break overlay obscures HUD elements at the screen edge

### DISCUSSION THREAD

**Discussion by sineplusx** [ppy/osu#29788] [Q&A] [answered]
### Type

Cosmetic

### Bug description

Title and images should be pretty self-explanatory.

Edit: Removed unrelated opinion about the shadow

### Screenshots or videos

| Before update | Current version |
|--------|--------|
| ![](https://github.com/user-attachments/assets/655e7751-a10e-44fe-8229-d9dc1548aef4) | ![](https://github.com/user-attachments/assets/3b92b563-55ef-4df8-bbb4-1623af5783cf) |

### Version

2024.906.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16919361/compressed-logs.zip)

**Comment by SchiavoAnto** [comment]
I can reproduce this issue with both Argon and the skin you provided but only on that specific map. Other maps don't have this problem for me.

**Comment by peppy** [comment]
I think it looks pretty okay? Maybe we can tint it another colour to make it looks better.

**Reply by peppy** [reply]
It was pointed out that I missed the fact that the gradients are covering HUD elements now, which is likely a problem we want to address..

**Reply by sineplusx** [reply]
Bumping this because it has not been addressed yet. Would it be possible to turn this back into an issue for better tracking?

**Reply by SchiavoAnto** [reply]
If one of the devs confirms that this problem needs a fix, I can submit a PR.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/32342

**Comment by DarkGradient** [comment]
Well, now it's looks "flat".
Sad actually, but i hope devs will bring option to enable "Gradient" for break overlay.
